test/hotspot/jtreg/vmTestbase/nsk/share/Finalizable.java

```diff
@@ -22,29 +22,47 @@
  */
 
 package nsk.share;
+import java.lang.ref.Cleaner;
 
 /**
  * Finalizable interface allows <tt>Finalizer</tt> to perform finalization of an object.
  * Each object that requires finalization at VM shutdown time should implement this
- * interface and activate a <tt>Finalizer</tt> hook.
+ * interface and call the <tt>registerClenup</tt> to activate a <tt>Finalizer</tt> hook.
  *
  * @see Finalizer
  */
 public interface Finalizable {
 
+    /**
+     * This method will be implemented by FinalizableObject and is called in <tt>finalizeAtExit</tt>.
+     *
+     * @see Finalizer
+     */
+    public void cleanup();
+
     /**
      * This method will be invoked by <tt>Finalizer</tt> when virtual machine
      * shuts down.
      *
      * @throws Throwable if any throwable exception thrown during finalization
      */
-    public void finalizeAtExit() throws Throwable;
+    default public void finalizeAtExit() throws Throwable {
+        cleanup();
+    }
 
-    /*
-     * This method will register a cleanup method and creates an instance of Finalizer
+    /**
+     * This method will register a cleanup method and create an instance of Finalizer
      * to register the object for finalization at VM exit.
      * It is implemented in FinalizableObject.
```

Comment:
This sentence can go since it's implemented here now.

Reply:
Done.